modules/users/client/components/InterfaceLanguagePanel.component.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import LanguageSwitch from '@/modules/core/client/components/LanguageSwitch.component';
+import LanguageSwitch from '@/modules/core/client/components/LanguageSwitch';
```

Comment:
This component isn't used anymore in Angular templates, so we can remove `.component.js` part in the file — thus build won't turn it into Angular component anymore.